embedchain/loaders/audio.py

```diff
@@ -0,0 +1,33 @@
+import hashlib
+
+from embedchain.helpers.json_serializable import register_deserializable
+from embedchain.loaders.base_loader import BaseLoader
+
+
+@register_deserializable
+class AudioLoader(BaseLoader):
+    def load_data(self, url):
+        """Load data from an Audio file."""
+        try:
+            import whisper
+        except ImportError:
+            raise ImportError(
+                'Audio requires extra dependencies. Install with `pip install --upgrade "embedchain[audio]"`'
+            ) from None
+        
+        model = whisper.load_model("base.en")
```

Comment:
Consider defaulting to "base.en", but making the mode selection configurable (i.e. it can be passed in to the `load_data`)